card/VERSION

```diff
@@ -1 +1 @@
-1.14.8s
+1.15.pre
```

Comment:
Interesting.  Is there a spec for that?

Then we want any version reference in all three gems to reference (however indirected) card/VERSION ?